scripts/launch-ci-e2e.sh

```diff
@@ -18,6 +18,9 @@ set -euo pipefail
 
 helm version
 go version
+
+make manager
+
```

Comment:
One of the points of a "clean slate" is we will finally be able to get rid of this launch script. The new e2e2 tests do not use this script and the make file rules already ensure to build the manager before launching the test.

If we take this approach, we will have to remember later that this script was ONLY required for the old ways.